Network: Return ACL logs from syslogs when the OVN controller is deployed in MicroOVN (#14327)
canonical/lxd: doc/howto/network_ovn_setup.md

       lxc config set network.ovn.northbound_connection <ovn-northd-nb-db>

   ```{note}

>    If you are using a MicroOVN deployment, you can pass the value of the MicroOVN node IP address you want target prefixed with `ssl:` and suffixed with the `:6641` port number that should target the OVN central service within MicroOVN.

Reviewer: ```suggestion
   If you are using a MicroOVN deployment, pass the value of the MicroOVN node IP address you want to target. Prefix the IP address with `ssl:`, and suffix it with the `:6641` port number that corresponds to the OVN central service within MicroOVN.
```

Author: fixed
